test_scripts/Smoke/Resumption/ATF_Resumption_big_amount_of_data.lua

```diff
@@ -0,0 +1,185 @@
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+
+function Test:StartSDL_With_One_Activated_App()
+  self:runSDL()
+  commonFunctions:waitForSDLStart(self):Do(function()
+    self:initHMI():Do(function()
+      commonFunctions:userPrint(35, "HMI initialized")
+      self:initHMI_onReady():Do(function ()
+        commonFunctions:userPrint(35, "HMI is ready")
+        self:connectMobile():Do(function ()
+          commonFunctions:userPrint(35, "Mobile Connected")
+          self:startSession():Do(function ()
+            commonFunctions:userPrint(35, "App is registered")
+            default_app = self.applications[default_app_params.appName]
+            commonSteps:ActivateAppInSpecificLevel(self, default_app)
+            EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL",audioStreamingState = "AUDIBLE", systemContext = "MAIN"})
+            commonFunctions:userPrint(35, "App is activated")
+          end)
+        end)
+      end)
+    end)
+  end)
+end
+
+function Test:AddCommand()
+  for i = 1, 20 do
+    self.mobileSession:SendRPC("AddCommand", { cmdID = i, vrCommands = {"VRCommand" .. tostring(i)}})
+  end
+  local on_hmi_call = EXPECT_HMICALL("VR.AddCommand"):Times(20)
+  on_hmi_call:Do(function(_, data)
+    self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  EXPECT_RESPONSE("AddCommand", { success = true, resultCode = "SUCCESS" }):Times(20)
+  EXPECT_NOTIFICATION("OnHashChange"):Do(function(_, data)
+    self.currentHashID = data.payload.hashID
+  end):Times(20)
+end
+
+function Test:AddSubMenu()
+  for i = 1, 20 do
+    self.mobileSession:SendRPC("AddSubMenu", { menuID = i, position = 500,
+                menuName = "SubMenu" .. tostring(i)})
+  end
+  local on_hmi_call = EXPECT_HMICALL("UI.AddSubMenu"):Times(20)
+  on_hmi_call:Do(function(_, data)
+    self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  EXPECT_RESPONSE("AddSubMenu", { success = true, resultCode = "SUCCESS" }):Times(20)
+  EXPECT_NOTIFICATION("OnHashChange"):Do(function(_, data)
+    self.currentHashID = data.payload.hashID
+  end):Times(20)
+end
+
+function Test:AddChoiceSet()
+  for i = 1, 20 do
+    self.mobileSession:SendRPC("CreateInteractionChoiceSet", {interactionChoiceSetID = i,
+        choiceSet = { { choiceID = i, menuName = "Choice" .. tostring(i), vrCommands = { "VrChoice" .. tostring(i)}}}})
+  end
+    local on_hmi_call = EXPECT_HMICALL("VR.AddCommand"):Times(20)
+    on_hmi_call:Do(function(_,data)
+      grammar_ID= data.params.grammarID
+      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+    EXPECT_RESPONSE("CreateInteractionChoiceSet", { success = true, resultCode = "SUCCESS" }):Times(20)
+    EXPECT_NOTIFICATION("OnHashChange"):Do(function(_, data)
+      self.currentHashID = data.payload.hashID
+    end):Times(20)
+end
+
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Transport unexpected disconnect. Media app resume at FULL level")
```

Comment:
@LitvinenkoIra, I think that this test should be not applicable only for media app, but if you leave you should be sure that application is media with:
config.application1.registerAppInterfaceParams.isMediaApplication = true